### PR TITLE
mgr/dashboard: Downstream branding

### PR DESCRIPTION
**Pull request by Exotelis**
Adds the downstream branding to the ceph dashboard.

- New sign in page
- New about dialogue
- Changed color scheme
- Replaced logos
- Updated doc URLs

### CODE REVIEW

File: src/pybind/mgr/dashboard/frontend/src/app/core/layouts/login-layout/login-layout.component.brand.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  </section>
+
+  <footer>
+    <div class="container">
+      <div class="row">
+        <div class="col-sm-9">
+          <span>© 2020 Ceph contributors. Free software (LGPL 2.1)</span>
+        </div>
+        <div class="col-sm-3 text-right d-none d-sm-block">
+          <img alt="SUSE Logo" src="assets/SUSE_Chameleon_L_White-neg_sRGB-250.png" class="footer-logo">
```
Comment (smithfarm) [inline src/pybind/mgr/dashboard/frontend/src/app/core/layouts/login-layout/login-layout.component.brand.html:47]: Seems to cause linter error:

"src/app/core/layouts/login-layout/login-layout.component.brand.html: line 47, col 11, attributes for one tag on the one line should be limited to 1"

**Comment by smithfarm** [comment]
@Exotelis Could you please sign your commit? `git commit --amend --signoff`. Thanks!

**Comment by Exotelis** [comment]
> @Exotelis Could you please sign your commit? `git commit --amend --signoff`. Thanks!

Thank you for pointing this out, updated the commit :+1:

**Comment by smithfarm** [comment]
I cherry-picked this into ses7 and squashed it into the previous dashboard branding commit. Thanks!

**Comment by Exotelis** [comment]
With the last updated I fixed BZ #1175120

**Comment by smithfarm** [comment]
@Exotelis I cherry-picked f980b1f into ses7 and squashed it into the previous dashboard branding commit. Thanks!

**Comment by Exotelis** [comment]
This pr will be closed and a new one will be opened that points directly on the ses7 branch.

**Pull request closed without merging**